packages/mui-material/src/AvatarGroup/AvatarGroup.js

```diff
@@ -184,6 +191,10 @@ AvatarGroup.propTypes /* remove-proptypes */ = {
     PropTypes.func,
     PropTypes.object,
   ]),
+  /**
+   * Specify total amount of avatars.
```

Comment:
I'm not sure if `children.length` work but it is nice to document it somewhere that it uses children.length as default.
```suggestion
   * Specify total amount of avatars.
   * @default children.length
```